client/web/src/components/exit-node-selector.tsx

```diff
@@ -84,6 +86,7 @@ export default function ExitNodeSelector({
             "bg-indigo-500 hover:bg-indigo-400": using,
           })}
           onClick={() => setOpen(!open)}
+          disabled={disabled}
```

Comment:
I think we should possibly remove the button entirely rather than just disabling it?  Just make it plain text information indicating what exit node is active?  Currently, the mouse cursor still changes to indicate you can click, but doing so does nothing, which is a weird experience.

Reply:
For now I threw the `cursor-not-allowed` class on the disabled buttons, can tweak in a follow-up.